package/dataset-kitti-full/install.sh

```diff
@@ -54,6 +54,12 @@ if [ "${?}" != "0" ] ; then
 fi
 rm ${KITTI_LABELS_ARCHIVE}
 
+mkdir data_object_image_2
+mkdir data_object_image_2/training
+cd data_object_image_2/training
+ln -s ../../training/image_2 image_2  
+## training/image_2/ data_object_image_2/training/image_2
```

Comment:
Move two lines above, and say that that's the intent but it doesn't work that way :)